app/scripts/modules/kubernetes/src/v2/pipelines/stages/deployManifest/manifestStatus.component.ts

```diff
@@ -0,0 +1,65 @@
+import { IComponentOptions, IController, IScope, module } from 'angular';
+import { KubernetesManifestService } from '../../../manifest/manifest.service';
+
+class KubernetesDeployManifestManifestStatusCtrl implements IController {
+  public manifestContents: any;
+  public stage: any;
+  public application: any;
+  public statuses: string;
+
+  private _manifest: any;
+
+  public get manifest() {
+    return this._manifest;
+  }
+
+  public set manifest(manifest) {
+    this._manifest = manifest;
+    this.statuses = '';
+    if (manifest.status == null) {
+      return;
+    }
+    const statuses = Object.keys(manifest.status).reduce((arr, key) => {
+      if (manifest.status[key].state) {
+        arr.push(key);
+      }
+      return arr;
+    }, []);
+    if (statuses.length > 0) {
+      this.statuses = '(' + statuses.join(', ') + ')';
+    }
+  }
+
+  constructor(private kubernetesManifestService: KubernetesManifestService, private $scope: IScope) {
+    'ngInject';
+
+    this.kubernetesManifestService.makeManifestRefresher(
+      this.application,
+      this.$scope,
+      {
+        account: this.stage.context.account,
+        location: this.manifestContents.metadata.namespace,
+        name: this.manifestContents.kind.toLowerCase() + ' ' + this.manifestContents.metadata.name,
+      },
+      this,
+    );
+  }
+}
+
+class KubernetesDeployManifestManifestStatusComponent implements IComponentOptions {
+  public bindings: any = { manifestContents: '<', stage: '<', application: '<' };
+  public controller: any = KubernetesDeployManifestManifestStatusCtrl;
+  public controllerAs = 'ctrl';
+  public template = `
+    <span ng-if="ctrl.statuses">
+      {{ ctrl.statuses }}
```

Comment:
Can we add some sort of visual indication to make this easier to scan? Something in 
![image](https://user-images.githubusercontent.com/4874941/39549546-3e148dde-4e2c-11e8-8550-b86a9f79d714.png) 

from the styleguide can help

Reply:
Also, most of these register messages in the condition that we could either show inline, or in a hover-popup style detail to help give context as to why something is/isn't stable.

Reply:
done